Sources/Client/MumbleLink.h

```diff
@@ -0,0 +1,41 @@
+#include "Player.h"
```

Comment:
You should put a header guard at the beginning of the header:
```cpp
#pragma once

...
```
And wrapping the classes into the `spades` namespace for consistency is probably a good idea.